ui/lib/pki/addon/components/pki-key-parameters.js

```diff
@@ -20,7 +20,7 @@ const KEY_BITS_OPTIONS = {
   any: [0],
 };
 
-export default class KeyParameters extends Component {
+export default class PkiKeyParameters extends Component {
```

Comment:
@hashishaw and I had discussed nesting these components in a `pki/` folder so that there weren't any conflicts with the addon engine. But for some reason the role form had trouble finding `<Pki::KeyParameters>` so I opted for prefixing the files with `pki-` instead